script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -422,7 +436,7 @@ Perhaps regaining consciousness as a result of my call, Arcueid weakly opens her
 [sha:d07dee620a99bc5bc4f1a24f14403a6748ecd24a]{
 -- Page 584, Offset 10190.
 -- 「な、なんでここにいるの!?　ダメ、逃げて……！」
-"W-What are you doing here? No, run away...!"
+"W-What are you doing here? You need to...get out of here...!"
```

Comment:
I'm not good with punctuation so just checking on this. Should there be a space after the first ellipses?